src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -161,6 +165,44 @@ describe('manage nsfs cli bucket flow', () => {
             path_exists = await is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
             expect(path_exists).toBe(false);
         });
+
+        it('cli delete bucket not fail with force flag, when bucket path is not empty', async () => {
+            let action = ACTIONS.ADD;
+            const bucket_options = { name: "bucket4", owner: bucket_defaults.owner, path: bucket_defaults.path };
+            // write the json_file_options
+            const path_to_option_json_file_name = await create_json_bucket_options(path_to_json_bucket_options_dir, bucket_options);
+            const command_flags = {config_root, from_file: path_to_option_json_file_name};
+            // create the bucket and check the details
+            await exec_manage_cli(TYPES.BUCKET, action, command_flags);
+            const bucket = await read_config_file(config_root, CONFIG_SUBDIRS.BUCKETS, "bucket4");
+            assert_bucket(bucket, bucket_options);
+            //here a dummy file is creating in bucket storage location(bucket_defaults.path), 
+            //using existing config file creation method(create_json_bucket_options) and bucket options code for this
+            await create_json_bucket_options(bucket_defaults.path, bucket_options);
+            const delet_bucket_options = { config_root, name: "bucket4", force: true};
+            action = ACTIONS.DELETE;
+            const resp = await exec_manage_cli(TYPES.BUCKET, action, delet_bucket_options);
+            expect(JSON.parse(resp.trim()).response.code).toBe(ManageCLIResponse.BucketDeleted.code);
```

Comment:
What about checking that the folder doesn't exist anymore? (like in "cli delete bucket and delete temp dir")
```
            path_exists = await is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
            expect(path_exists).toBe(false)
```

Reply:
We do not delete the folder in cli delete flow, we can check for schema file exist or not along with existing check

Reply:
@naveenpaul1  I'm not sure I understand this, I copied it from an existing test, would you please explain it in more detail?

Reply:
updated the code , code will check for the bucket schema

> const config_path = path.join(config_root, CONFIG_SUBDIRS.BUCKETS, bucket_five + '.json');
> await fs_utils.file_must_not_exist(config_path);

Reply:
Deleting temp dir is covered in different test case, why we need to add that scenario here again?

Reply:
No need to, if you think you already covered both things, then it is fine just check the response (as you did and remove the additional check of the config if you think it is already covered).

Reply:
no issues, I will keep both checks